index/scorch/scorch.go

```diff
@@ -507,8 +533,9 @@ func (s *Scorch) StatsMap() map[string]interface{} {
 	m["num_items_introduced"] = m["TotIntroducedItems"]
 	m["num_items_persisted"] = m["TotPersistedItems"]
 	m["num_recs_to_persist"] = m["TotItemsToPersist"]
-	m["num_bytes_used_disk"] = m["CurOnDiskBytes"]
-	m["num_files_on_disk"] = m["CurOnDiskFiles"]
+	m["num_bytes_used_disk"] = numBytesUsedDisk
```

Comment:
According to the comments in this section, some of the stats have 2 names for backwards compatibility.  But if that is the case, I don't think we should keep introducing multiple names for each stat.  Or is there more to this than I understand?

Reply:
agree..i was just following the pattern there. So m["CurOnDiskBytesByRoot"]  isn't needed here.

Reply:
OK, so it will be removed?

Reply:
which one to be removed now? m["CurOnDiskBytesByRoot"]  is removed already if you thats what you were referring to.